src/pybamm/models/submodels/equivalent_circuit_elements/rc_element.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, param, element_number, options=None):
 
     def get_fundamental_variables(self):
         vrc = pybamm.Variable(f"Element-{self.element_number} overpotential [V]")
+        vrc.print_name = "eta"
```

Comment:
If we are doing this with latex, eta can also be put in as the symbol, but I have not looked at how consistent this is in the codebase

Reply:
Yes, these are in `GREEK_LETTERS` and are handled at the time.